app/jobs/scheduled/encrypted_post_timer_evaluator.rb

```diff
@@ -8,10 +8,11 @@ def execute(args)
       EncryptedPostTimer.pending.find_each do |encrypted_post_timer|
         ActiveRecord::Base.transaction do
           encrypted_post_timer.touch(:destroyed_at)
-          next if posts_to_delete(encrypted_post_timer).blank?
+          posts_to_delete = posts_to_delete(encrypted_post_timer)
```

Comment:
I think bad things will happen if you name the variable the same thing as a method :) Maybe the method could be `find_posts_to_delete`?